vms/platformvm/vm.go

```diff
@@ -205,7 +205,10 @@ func (vm *VM) Initialize(
 		chainCtx.Log,
 		chainCtx.NodeID,
 		chainCtx.SubnetID,
-		chainCtx.ValidatorState,
+		validators.NewLockedState(
```

Comment:
Are there other VMs that provide the raw state here?

Probably makes sense to double-check C/X-chain initialization.

Reply:
This is a P-chain specific bug. In the C-chain and the X-chain the ctx.ValidatorState is already locked.